src/plugins/formulas/__tests__/formulas.spec.js

```diff
@@ -2356,4 +2218,28 @@ describe('Formulas general', () => {
       });
     });
   });
+
+  it('should work with a sheet name that contains a `-` (#8057)', () => {
+    handsontable({
+      data: [
+        [1, 2, 3, 4, 5],
+        [9, 8, 7, 6, '=myOwnCalc'],
+      ],
+      formulas: {
+        engine: HyperFormula,
+        sheetName: 'my-sheet',
+        namedExpressions: [
+          {
+            name: 'myOwnCalc',
+            expression: '=\'my-sheet\'!$A$1+100',
```

Comment:
That's not the case from #8057. Quoted sheet names are correctly handled and tested by HF. We were discussion better handling misuse scenario:

```suggestion
            expression: '=my-sheet!$A$1+100',
```

Reply:
Rewritten this test case to check if this setup crashes the table in https://github.com/handsontable/handsontable/pull/8669/commits/c52a4f85f0966f7c2eeed8e267a374aff3480695.